### PR TITLE
Issue #14436: Enforced naming convention for MissingCtor

### PR DESCRIPTION
**Pull request by CubeTures**
Issue #14436

### CODE REVIEW

**Comment by CubeTures** [comment]
Messed up by closing the other PR instead of just commenting I needed to fix it, so here is the actual fix to the problem I had.

**Comment by sktpy** [comment]
> Looks like I still have CI issues though?

Failing checks in CI are unrelated.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingCtorCheckTest.java
Comment (MANISH-K-07) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingCtorCheckTest.java:37]: The naming suits the input files and test methods....

**Review by MANISH-K-07 — COMMENTED** [review]
Everything lgtm....
Thanks a lot for the update

**Review by MANISH-K-07 — COMMENTED** [review]
Please wait for the approval of maintainers

**Review by nrmancuso — APPROVED** [review]
@CubeTures thanks for your contribution! Good to merge as GitHub CI passes

**Comment by nrmancuso** [comment]
CI failures are not related to changes here

**Pull request merged**
